vendor.conf

```diff
@@ -1,4 +1,4 @@
-k8s.io/kubernetes a48f11c2257d84b0bec89864025508b0ef626b4f https://github.com/kubernetes/kubernetes
+k8s.io/kubernetes 7488d1c9210e60aef9ad49f07cb5d8a24152db88 https://github.com/kubernetes/kubernetes
 k8s.io/client-go master https://github.com/kubernetes/client-go
```

Comment:
very likely we need to bump everything related to k8s as well

Reply:
I did update most of these. Let us switch back to commit ids. I'll update the PR.